web-registry/src/components/organisms/BastketDetailsActionsBar/BastketDetailsActionsBar.tsx

```diff
@@ -0,0 +1,54 @@
+import { Box } from '@mui/material';
+
+import ContainedButton from 'web-components/lib/components/buttons/ContainedButton';
+import CurrentCreditsIcon from 'web-components/lib/components/icons/CurrentCreditsIcon';
+import { StickyBar } from 'web-components/lib/components/sticky-bar/StickyBar';
+import InfoTooltipWithIcon from 'web-components/lib/components/tooltip/InfoTooltipWithIcon';
+
+import { Link } from 'components/atoms';
+
+import {
+  BUY_BASKET_TOKENS,
+  BUY_BASKET_TOKENS_TOOLTIP,
+  BUY_BUTTON_LINK,
+} from './BastketDetailsActionsBar.constants';
+
+type Params = {
```

Comment:
minor nit: shouldn’t this be
```suggestion
type Props = {
```
?